Automattic/jetpack
# E2E Tests: Use headless connection for pre-connection

#### Changes proposed in this Pull Request:

* Use headless connection via JP Start APIs in setup stage
* Switch to In-place connection flow for Plugin Updater test due to 1156386383419466-as-1200246181748957/f

#### Jetpack product discussion


1156386383419466-as-1200005920011672

#### Does this pull request change what data or activity we track or use?


n/a

#### Testing instructions:






* Make sure CI is green
* Run the suite locally, make sure the initial connection happens headleslly
*
